File: Arrays/Day3_ReverseArray/Day3_reverseArray_InbuiltFunctionMethods.cpp
// C++ Program to reverse an array using inbuilt methods

/*
You are given an array of integers arr[]. Your task is to reverse the given array.
Note: Modify the array in place.
*/


#include <iostream>
#include <vector>
#include <algorithm> 
using namespace std;

// function to reverse an array
void reverseArray(vector<int> &arr) {
    reverse(arr.begin(), arr.end());
}

int main() {
    vector<int> arr = { 1, 4, 3, 2, 6, 5 };

    reverseArray(arr);
    for(int i = 0; i < arr.size(); i++) 
        cout << arr[i] << " ";
    return 0;
}

File: Arrays/Day3_ReverseArray/Day3_reverseArray_InbuiltFunctionMethods.py
'''
You are given an array of integers arr[]. Your task is to reverse the given array.
Note: Modify the array in place.
'''

# Python Program to reverse an array using inbuilt methods

# function to reverse an array
def reverse_array(arr):
    arr.reverse()

if __name__ == "__main__":
    arr = [1, 4, 3, 2, 6, 5]

    reverse_array(arr)
    print(" ".join(map(str, arr)))